server/src/main/java/io/druid/server/QueryLifecycle.java

```diff
@@ -112,16 +112,19 @@ public QueryLifecycle(
    * either fully iterated or throws an exception.
    *
    * @param query             the query
-   * @param authorizationInfo authorization info from the request; or null if none is present. This must be non-null
-   *                          if security is enabled, or the request will be considered unauthorized.
+   * @param user              authentication token from the request
+   * @param namespace         authentication namespace of the request
```

Comment:
Some of these parameters don't exist, please make sure the javadoc is up to date.